source/deploy/postgres-migration-assist-tool.rst

```diff
@@ -81,95 +100,100 @@ Step 4 - Run pgloader
 
    pgloader migration.load > migration.log
 
-Carefully read the log file to analyze whether there were any errors during the migration process. If there were any errors, please contact Mattermost for further guidance.
+Carefully review `migration.log` for errors (e.g., duplicate-key or missing-table warnings). Use the `mattermost/pgloader:latest` Docker image to avoid build/auth issues.
```

Comment:
```suggestion
Carefully review ``migration.log`` for errors (e.g., duplicate-key or missing-table warnings). Use the ``mattermost/pgloader:latest`` Docker image to avoid build/auth issues.
```